### ISSUE TITLE
Error showing `Chains` 

### ISSUE THREAD

**Issue by mohamed82008** [TuringLang/MCMCChains.jl#58]
This code gives an error displaying `Chains` with Turing and MCMCChains master.
```julia
using Turing
@model gdemo(x, y) = begin
   s ~ InverseGamma(2,3)
   m ~ Normal(0,sqrt(s))
   x ~ Normal(m, sqrt(s))
   y ~ Normal(m, sqrt(s))
end
sample(gdemo(2.0, 1.5), SGLD(100, 0.01))
```

**Comment by xukai92** [comment]
You can't display chain less than 200 samples

**Comment by mohamed82008** [comment]
Why is that so? Seems like there shouldn't be any problem displaying any struct in MCMCChains. Why is the error behavior better than conditioning on the number of samples and avoiding the error by displaying something reasonable?

**Comment by xukai92** [comment]
Sorry I closed it by mistake. I agree. By my previous reply I was simply trying to put a note on why this happens (I.e. it's a MCMCChain display issue). We need to display something reasonable.

**Comment by xukai92** [comment]
If I remember correctly it's something related to functions that computes stats refuses to do so for <200 samples

**Comment by cpfiffer** [comment]
It's the `msce_bm` function, and I was just working on that last night. I should have something soon to address this.

**Comment by cpfiffer** [comment]
Closed by #59.
